docs/README.md

```diff
@@ -172,6 +174,23 @@ CORECLR_PROFILER_PATH_64
 OTEL_DOTNET_AUTO_INTEGRATIONS_FILE
 ```
 
+You can also use the [instrument.sh](../instrument.sh) script which uses following
+environment variables as parameters:
+
+| Parameter          | Description                                                            | Required | Default value        |
+|--------------------|------------------------------------------------------------------------|----------|----------------------|
+| `DISTRIBUTION`     | Possible values: `linux-glibc`, `linux-musl`, `macos`, `windows`       | Yes      |                      |
+| `ENABLE_PROFILING` | Whether to set the .NET CLR Profiler, possible values: `true`, `false` | No       | `true`               |
+| `INSTALL_DIR`      | Location where binaries are to be installed                            | No       | `./otel-dotnet-auto` |
+
+> On macOS [`coreutils`](https://formulae.brew.sh/formula/coreutils) is required.
+
+```sh
+curl -fL https://raw.githubusercontent.com/open-telemetry/opentelemetry-dotnet-instrumentation/main/instrument.sh -O
```

Comment:
I think this should point to the release artifact instead of to the actual source. Specially when main is not a release tree.

Reply:
I am not sure as any `Writer` (approver) can change/remove/add release artifacts. The `main` branch is extremely protected - requires GPG signatures, also only the maintainers can merge/push to `main. We can consider changing it to a "release tag" during the release process. WDYT?

Reply:
@RassK PTAL https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1357